docs/operate/get-started/other-hardware/manage-modules.md

```diff
@@ -322,66 +208,71 @@ For guidance on configuring the other parameters, see the documentation for each
+- [C++](https://github.com/viamrobotics/module-example-cpp)
 
-{{% /tab %}}
-{{< /tabs >}}
+  {{% /tab %}}
+  {{< /tabs >}}
 
-4. Create an [organization API key](/dev/tools/cli/#create-an-organization-api-key) with the [owner](/manage/manage/rbac/) role, which the GitHub action will use to authenticate to the Viam platform, using one of the following methods:
+3. [Create an organization API key](/dev/tools/cli/#create-an-organization-api-key) with owner role:
 
-   - Use the Viam CLI to create an organization API key, which includes the owner role by default:
+   ```sh {class="command-line" data-prompt="$"}
+   viam organizations api-key create --org-id <org-id> --name <key-name>
+   ```
 
-     ```sh {class="command-line" data-prompt="$"}
-     viam organizations api-key create --org-id <org-id> --name <key-name>
-     ```
+1. Add the key ID and value as GitHub repository secrets named `viam_key_id` and `viam_key_value`.
 
-   - Use the organizations page on the [Viam app](https://app.viam.com/) to generate a new organization API key.
-     Make sure your organization API key is set to **Role: Owner**, or the GitHub action will not be able to successfully authenticate during runs.
-     If you are using an existing organization API key which is not set to **Role: Owner**, you can change an API key's permissions from the Viam app on the organizations page by clicking the **Show details** link next to your API key.
-     The operator role cannot be used to authenticate GitHub action runs.
-     For more information see [Organize your machines](/manage/reference/organize/).
+1. Push a tag or create a [release](https://docs.github.com/en/repositories/releasing-projects-on-github) in GitHub to trigger the build.
+   Once the build is complete, the module will automatically update in the [Viam Registry](https://app.viam.com/registry), and the machines set to use the latest [version](/operate/reference/module-configuration/#module-versioning) of the module will automatically update to the new version.
 
-   Both methods return a `key id` and a `key value` which together comprise your organization API key.
+### Update manually
 
-1. Then, configure your GitHub repository to use your organization API key to authenticate during GitHub action runs, following the steps below:
+Use the [Viam CLI](/dev/tools/cli/) to manually update your module:
 
-   1. In the GitHub repository for your project, select **Settings**, then **Secrets and variables**, then **Actions**.
+1. Edit your module code and update the [`meta.json`](/operate/get-started/other-hardware/#metajson-reference) file if needed.
+   For example, if you've changed the module's functionality, update the description in the `meta.json` file.
 
-   1. Click the green **New repository secret** button, enter `viam_key_id` as the **NAME**, paste the value for `key id` from above into the **Secret** text field, then click **Add secret**.
+2. For Python modules only, package your files as an archive:
```

Comment:
Since this may not apply to what people need to do, it may be worth at least signposting that that may not be what they need to run.
```suggestion
2. For Python modules only, package your files as an archive, for example:
```